keps/sig-storage/1491-csi-migration-vsphere/README.md

```diff
@@ -35,6 +40,27 @@ used as as described in its parent KEP. For all other contents, please refer to
   on kubelet and treat vSphere CSI migration as already complete. And directly redirect traffic to CSI 
   driver for all vsphere volume related operations.
 
+## Design Details
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+### Prerequisite testing updates
+Additional tests added in the [vSphere CSI Driver repository](https://github.com/kubernetes-sigs/vsphere-csi-driver) to ensure vSphere CSI Migration is production ready.
+ - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/vcp_to_csi_create_delete.go
+ - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/vcp_to_csi_attach_detach.go
+ - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/vcp_to_csi_full_sync.go
+ - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/tests/e2e/vcp_to_csi_syncer.go
```

Comment:
Should these tests be in the e2e tests section?

Reply:
For prerequisites, you can probably just point to existing e2e tests in CSI driver and say that no additional prerequisite tests are needed.

Reply:
done.